web/src/page_params.ts

```diff
@@ -2,18 +2,21 @@ import $ from "jquery";
 
 const t1 = performance.now();
 export const page_params: {
+    development_environment: boolean;
     language_list: {
         code: string;
         locale: string;
         name: string;
         percent_translated: number | undefined;
     }[];
-    development_environment: boolean;
```

Comment:
I assume your intention here was to put this list in alphabetical order. If so, you did not succeed: `language_list` should go after `is_spectator`.